plugins/hangmanplugin.py

```diff
@@ -176,3 +179,21 @@ def getWord(api_key, minCorpus=6000, maxCorpus=-1, minLength=5, maxLength=20):
         jobj = res.read().decode('utf-8')
         jparse = json.loads(jobj)
         return jparse['word']
+
+    @staticmethod
+    def checkWord(api_key, minCorpus=6000, maxCorpus=-1, minLength=1, maxLength=-1):
```

Comment:
You should pass in another parameter here. `checkWord[0]` also doesn't exist in the scope of this method. Do something like this:
`def checkWord(api_key, word, minCorpus=6000, maxCorpus=-1, minLength=1, maxLength=-1):` instead, and then use the `word` parameter in the method body.